packages/@rnw-scripts/just-task/package.json

```diff
@@ -14,16 +14,25 @@
   "dependencies": {
     "@rnw-scripts/jest-e2e-config": "1.1.8",
     "@rnw-scripts/jest-unittest-config": "1.2.6",
-    "depcheck": "^1.4.1"
+    "depcheck": "^1.4.1",
+    "glob": "^7.1.6",
+    "just-scripts": "^1.3.3"
   },
   "devDependencies": {
-    "@types/node": "^14.14.22",
-    "just-scripts": "^1.3.3"
+    "@types/node": "^14.14.22"
   },
   "peerDependencies": {
-    "just-scripts": "^1.3.3"
+    "eslint": "*",
```

Comment:
I considered just setting these as dependencies, to avoid needing to declare versions everywhere. I think this would break VSCode integration though, so we have each package declare the version.